util/fipstools/delocate/delocate.peg.go

```diff
@@ -1,10 +1,14 @@
 package main
 
+// Code generated by /Users/shanggu/workplace/github/peg/peg delocate.peg DO NOT EDIT.
```

Comment:
- Would you like to leave your username here, Bryce?
- Also is there a reason why this file is changed so much?

Reply:
This is auto generated code. Will remove.